openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -0,0 +1,30 @@
+{% spaceless %}
+{% load i18n %}
+
+{% if file_upload_type %}
+    {% if header %}
+        <header class="{{ class_prefix }}__display__header">
+            <h3 class="{{ class_prefix }}__display__title">
+                {% trans header %}
```

Comment:
@andy-armstrong The strings will get translated here. We are only passing them from respective files.